plugin.xml

```diff
@@ -99,21 +99,21 @@
     <framework src="CoreTelephony.framework"/>
     <framework src="EventKit.framework"/>
     <framework src="EventKitUI.framework"/>
+    <framework src="MediaPlayer.framework"/>
     <framework src="MobileCoreServices.framework"/>
     <framework src="QuartzCore.framework"/>
     <framework src="Security.framework"/>
     <framework src="Social.framework"/>
     <framework src="StoreKit.framework"/>
+    <framework src="SystemConfiguration.framework"/>
     <framework src="iAd.framework"/>
 
     <framework src="libxml2.dylib"/>
+    <framework src="libsqlite3.dylib" /> 
```

Comment:
I've built and tested with Xcode 7. It seems like you can still link `.dylib` libraries, they still exist in the `/usr/lib` directory; I think they're still available to maintain backwards compatibility with previous Xcode projects.